### PR TITLE
Migrate to go1.12, deprecate vendor directory

### PR DESCRIPTION
**Pull request by harshavardhana**

### CODE REVIEW

**Comment by wlan0** [comment]
LGTM - Thanks for checking for xtended FS support - this should prevent any future test/CI debts.

**Comment by harshavardhana** [comment]
Windows build is fixed by fixing xattr package upstream and `go.mod` is updated as well @vadmeste PTAL

File: cmd/policy-main.go [hunk 1]
```diff
@@ -444,13 +444,13 @@ func mainPolicy(ctx *cli.Context) error {
...
-		runPolicyListCmd(ctx)
+		runPolicyListCmd(ctx.Args().Tail())
 	case "links":
 		// policy links alias/bucket/prefix
-		runPolicyLinksCmd(ctx)
+		runPolicyLinksCmd(ctx.Args().Tail(), ctx.Bool("recursive"))
 	default:
 		// policy [download|upload|public|] alias/bucket/prefix
-		runPolicyCmd(ctx)
+		runPolicyCmd(ctx.Args().Tail())
```
Comment (vadmeste) [inline cmd/policy-main.go]: You should not Tail() this, the first argument is the access permission (download, upload, public)

**Comment by harshavardhana** [comment]
> I am not sure about mutiple `go: finding` in each make install command

That is upto `go` to decide. I don't know why it does that. But its not an issue @vadmeste

File: cmd/policy-main.go (hunk 1 above)
Comment (harshavardhana) [reply cmd/policy-main.go]: Fixed

**Pull request merged**